_sass/scaffolds.scss

```diff
@@ -303,6 +316,9 @@ li.sticky-nav{
   .project-banner-wrapper {
     flex-direction: column;
   }
+  .project-banner-separator {
```

Comment:
yank this (added above)

Reply:
i'd rather not, as it fixes my preferred changes per what you've added above.

Reply:
wait, what? nooo. okay, so, consider these two chunkz of css which result in the same thing:

```css
.project-banner-separator {
  top: 0;
}
.project-banner-separator {
  top: 666px;
}
.project-banner-separator {
  top: -32em;
}
.project-banner-separator {
  top: -3em;
}
```
...and
```css
.project-banner-separator {
  top: -3em;
}
```

the first example is like what we have now, which is hard to reason about. why is this getting defined and then overridden a short while later? 

what's your reasoning about this? are you saying that you want `top: 0`? or `top: -3em`? pick one and just define it once in the `.project-banner-separator` block above. yeah?

Reply:
bahhh, i'm dumb and didn't read this good. i understand the @media query now. ssssssoooorrrry